### ISSUE TITLE
Zoom to node with scale option

### ISSUE THREAD

**Issue by neolution** [bumbeishvili/org-chart#82]
First: You did a great job. Your org chart is one of the best I have seen 👍

My problem: I want to zoom to a node (nodeId). I know the „setCentered“-function, but I don‘t know how to combine it with a fix scale parameter. How can I do this?

By the way: I modified your zoomBehaviour with the d3.interpolateZoom-function with rho-option for better animation experience….perhaps you can integrate this in your code.

**Comment by bumbeishvili** [comment]
You can do it like this

```javascript
chart.setCentered("O-6162").initialZoom(1).render()
```

[Example](https://stackblitz.com/edit/web-platform-xs8dz1)


https://user-images.githubusercontent.com/6873202/131723223-1a26b8e1-e736-4ab5-8157-c51edd72144c.mp4




> By the way: I modified your zoomBehaviour with the d3.interpolateZoom-function with rho-option for better animation experience….perhaps you can integrate this in your code.

Curious, what advantages are there? And what are the differences between current and your implementation?

**Comment by neolution** [comment]
Thank you for your fast answer.

zoomOnNode:  Looks good. Thank you !
zoomBehaviour:

I extended your d3-const with

`interpolateZoom: d3Zoom.interpolateZoom,..`


After that, I modified your "behaviours.zoom" with:
`behaviors.zoom = d3.zoom().interpolate(d3.interpolateZoom.rho(2)) // 0.9 for less curvature
                    .on('zoom', (event, d) => this.zoomed(event, d)).scaleExtent([0.001, 20]);
                attrs.zoomBehavior = behaviors.zoom;`

I like the animation when I use "setCentered"-function with an automatic smooth zoomOut and zoomIn animation. Perhaps you have a better idea.
